sale_report_delivered_subtotal/README.rst

```diff
@@ -0,0 +1,95 @@
+==============================
+Sale Report Delivered subtotal
+==============================
+
+.. 
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! This file is generated by oca-gen-addon-readme !!
+   !! changes will be overwritten.                   !!
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+   !! source digest: sha256:e93ab1be2ad0674e3420e5cf018318bfb219b024f4c6da35bd15649537e93255
+   !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
+
+.. |badge1| image:: https://img.shields.io/badge/maturity-Production%2FStable-green.png
+    :target: https://odoo-community.org/page/development-status
+    :alt: Production/Stable
+.. |badge2| image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+.. |badge3| image:: https://img.shields.io/badge/github-OCA%2Fsale--reporting-lightgray.png?logo=github
+    :target: https://github.com/OCA/sale-reporting/tree/16.0/sale_report_delivered_subtotal
+    :alt: OCA/sale-reporting
+.. |badge4| image:: https://img.shields.io/badge/weblate-Translate%20me-F47D42.png
+    :target: https://translation.odoo-community.org/projects/sale-reporting-16-0/sale-reporting-16-0-sale_report_delivered_subtotal
+    :alt: Translate me on Weblate
+.. |badge5| image:: https://img.shields.io/badge/runboat-Try%20me-875A7B.png
+    :target: https://runboat.odoo-community.org/webui/builds.html?repo=OCA/sale-reporting&target_branch=16.0
+    :alt: Try me on Runboat
+
+|badge1| |badge2| |badge3| |badge4| |badge5|
+
+Adds subtotal price based on the delivered quantities field to the
+*Sales Report*.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Usage
+=====
+
+Go to *Sales > Reporting > Sales* and you'll find *Subtotal Delivered* and *Gross Weight
```

Comment:
```suggestion
Go to *Sales > Reporting* and you'll find *Untaxed Total Delivered* and *Gross Weight
```

Reply:
Done, thanks for the review

Reply:
@Yadier-Tecnativa this change should be put on the `readme` folder, not here.

Reply:
Done in readme folder